### DISCUSSION TITLE
分享的链接能否添加预览功能

### DISCUSSION THREAD

**Discussion by likepoems** [AlistGo/alist#785] [Ideas]
在图片，css，js等类型的文件，在复制直链后打开是默认提示下载的，能否添加个预览功能，可以在页面上看到内容后在选择是否保存

**Comment by xhofe** [comment]
直链就是用来下载的，你要预览直接复制地址栏里的链接啊。。。

**Reply by likepoems** [reply]
我的意思是css，js，mp3，jpg，png等格式文件可否支持鼠标右键在新标签页打开，alist右键打开后就直接下载了

**Reply by xhofe** [reply]
这取决于存储提供商，而不是由本程序控制 你可以了解一下https://developer.mozilla.org/zh-CN/docs/Web/HTTP/Headers/Content-Disposition

**Comment by ihipop** [comment]
但是目前看程序是做了强制下载的，我给了个 存储是本地的的 index.html 但是程序还是强制给下载 @Xhofe

**Reply by NovaShen0724** [reply]
2023.1.2没有更新此功能

**Reply by luckydzp** [reply]
确实直链会直接下载，而不能在线播放或者预览，可能需要在直链响应的response中，将Content-Disposition: attach改为Content-Disposition: inline，或者作者能否开放一下自定义http header的功能。

**Comment by luckydzp** [comment]
确实直链会直接下载，而不能在线播放或者预览，可能需要在直链响应的response中，将Content-Disposition: attach改为Content-Disposition: inline，或者作者能否开放一下自定义http header的功能。

**Comment by jeffon1996** [comment]
希望文本的分享链接可以加一个选项预览，类似[Link](https://raw.githubusercontent.com/AlistGo/alist/refs/heads/main/Dockerfile)这样，filerun有这样的选项
